test/parallel/test-dns-regress-7070.js

```diff
@@ -4,5 +4,5 @@ const assert = require('assert');
 const dns = require('dns');
 
 // Should not raise assertion error. Issue #7070
-assert.throws(() => dns.resolveNs([])); // bad name
-assert.throws(() => dns.resolveNs('')); // bad callback
+assert.throws(() => dns.resolveNs([]), /"name" argument must be a string/); // bad name
+assert.throws(() => dns.resolveNs(''), /"callback" argument must be a function/); // bad callback
```

Comment:
These lines are longer than 80 characters, which is something we want to avoid – could you move the second argument to a new line (and align that below the first argument)?